src/Mono.Android/Mono.Android.targets

```diff
@@ -42,6 +42,17 @@
     />
     <Touch Files="Android.Runtime\JNIEnv.g.cs" />
   </Target>
+  <Target Name="_BuildJniMarshalMethodsGen"
+      BeforeTargets="CoreCompile"
+      Inputs="$(JavaInteropFullPath)\tools\jnimarshalmethod-gen\Xamarin.Android.Tools.JniMarshalMethodGenerator.csproj"
+      Outputs="$(XAInstallPrefix)xbuild\Xamarin\Android\jnimarshalmethod-gen.exe;$(XAInstallPrefix)xbuild\Xamarin\Android\Java.Runtime.Environment.dll.config">
+    <MSBuild
+        Projects="$(JavaInteropFullPath)\tools\jnimarshalmethod-gen\Xamarin.Android.Tools.JniMarshalMethodGenerator.csproj"
+    />
+    <Exec
+        Command="sed -i .in.bak -e '/&lt;configuration&gt;/r $(MSBuildThisFileDirectory)\..\..\build-tools\scripts\java-interop.dllmap' $(XAInstallPrefix)xbuild\Xamarin\Android\Java.Runtime.Environment.dll.config"
```

Comment:
*This* part seems...okay...except for two things:

 1. This won't build on Windows. Can we instead use the `<ReplaceFileContents/>` task?
 2. The output file is in `$(XAInstallPrefix)xbuild\Xamarin\Android`. Why is this being generated in `Mono.Android.targets`, which places outputs into `$(XAInstallPrefix)xbuild-frameworks\...`?

If this belongs anywhere, it's in `src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets`.

Reply:
I have meanwhile moved it to src/monodroid/monodroid.targets which is hopefully good place too. We handle the configs at one place and the dllmaps use the monodroid shared libraries.

I can try to use `ReplaceFileContents` task. Hope it could be done as an addition to speedup the merge process.